### ISSUE TITLE
CLA details link missing in PR check list and incorrect status of CLA signing

### ISSUE THREAD

**Issue by hirenvadalia** [cla-assistant/cla-assistant#576]
URL to the linked Repo/Org:
https://github.com/openpbs/openpbs/pull/1730

Steps to reproduce the problem:
1. push or force-push new commits in new or existing PR

What is the expected result?
CLA should show as signed as I have already signed CLA. Also in PR check list it should show link to show details of CLA and its signing status.

What happens instead?
No link for details and status show as "Expected — Waiting for status to be reported"...
Sometime it works fine and shows expected results, but most of the time it fails.

## Optional
Recently we have moved our repo from PBSPro org to OpenPBS org and renamed repo to openpbs (earlier it was pbspro in PBSPro org). Also we still have PBSPro org and CLA on it as there are other repos in PBSPro which uses that CLA.

![Screen Shot 2020-06-02 at 6 09 54 PM](https://user-images.githubusercontent.com/17191391/83521102-5ac81f80-a4fc-11ea-96e9-b0790a190339.png)
![Screen Shot 2020-06-02 at 6 10 08 PM](https://user-images.githubusercontent.com/17191391/83521116-60256a00-a4fc-11ea-8e4d-60ef411fc2e5.png)

Browser/version:
Google Chrome: Version 83.0.4103.61 (Official Build) (64-bit)

Any other tested browsers/devices(OK/FAIL):

**Comment by KharitonOff** [comment]
Thanks for the notification! Can you let us know, when and in which PR this issue appears again, please?

**Comment by hirenvadalia** [comment]
@KharitonOff Here is one the https://github.com/openpbs/openpbs/pull/1784,
not sure author of that PR has signed new CLA or not, but still it it doesn't show details link, where user can sign CLA.

**Comment by tpai26** [comment]
> @KharitonOff Here is one the [openpbs/openpbs#1784](https://github.com/openpbs/openpbs/pull/1784),
> not sure author of that PR has signed new CLA or not, but still it it doesn't show details link, where user can sign CLA.

I have signed the new CLA, 
It got stuck again showing "Expected — Waiting for status to be reported", but using this passed it-  
https://cla-assistant.io/check/openpbs/openpbs?pullRequest=1784

**Comment by ibnipun10** [comment]
For me it is failing though I have signed it : https://github.com/lyft/amundsen/pull/487

**Comment by ibnipun10** [comment]
Configured my github and it worked

**Comment by hirenvadalia** [comment]
@KharitonOff This is another example of PR when CLA is stuck: https://github.com/openpbs/openpbs/pull/1811

Any ETA for fix to this issue?

**Comment by KharitonOff** [comment]
@ibakshay do you have any idea here? I can not find any log entries for this PR...

**Comment by ibakshay** [comment]
Hi @hirenvadalia, I retriggered the CLA check and CLA check is updated now for PR https://github.com/openpbs/openpbs/pull/1811. you can also retrigger the CLA check for a particular Pull Request using this API `https://cla-assistant.io/check/<orgname>/<reponame>?pullRequest=10`. 
In your case, the recheck API will be `https://cla-assistant.io/check/openpbs/openpbs?pullRequest=1811`
Since you are saying, this occurs very frequently to this repository, we shall keep this issue open to find the root cause for this issue. 

@KharitonOff, I also don't see the logs for this PR. Looks strange to me..

**Comment by subhasisb** [comment]
@hirenvadalia, all, I think i found the issue. What struck me was that the anchor icon on the cla-assistant site was showing tick-mark for gist, but x for webhook. So i compared with another repo where both were ticked. I manually created a webhook on our repo copying the url pattern and settings from our other repo, and boom, the status anchor changed to green, and CLA's are being auto reported.

So, the core of the issue was that when we install CLA-assistant, it did not create the webhook (or any other mechanism to intimate cla-assitant of PR events). We had noticed this earlier, but thought that this has probably changed to some other kind of integration (that does not need a webhook)...but now, looking at the anchor icon on cla-assistant page, it was clear that it still wanted a web-hook.

Why it silently failed to install a webhook - i have no idea about!
